Task01/NarcissisticNumbers.py

```diff
@@ -0,0 +1,34 @@
+import os,sys
+
+def main():
+    print_narcissistic_numbers(1,1001)
```

Comment:
```suggestion
    print_narcissistic_numbers(1, 1000)
```
тут я дал маху. Мы в функцию должны передавать числа в диапазоне которых мы ищем narcissistic numbers, а уже когда используем их в range - тогда и добавляем единичку, так как последнее число не включаеться в диапазон. поправлять нужно не тут, а в range

Reply:
в рандже я поправил, смотри